src/video/wayland/SDL_waylandwindow.c

```diff
@@ -812,6 +813,50 @@ void Wayland_HideWindow(_THIS, SDL_Window *window)
     }
 }
 
+static void
+handle_xdg_activation_done(void *data,
+                           struct xdg_activation_token_v1 *xdg_activation_token_v1,
+                           const char *token)
```

Comment:
The token is fetched but is unused. Call `xdg_activation_v1_activate` to raise the toplevel.

Reply:
Added in the latest revision, hopefully this is right...